addons/web/static/src/search/search_bar/search_bar.xml

```diff
@@ -1,7 +1,31 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
 
+    <t t-name="web.SearchBar.ActiveWebsiteFacet">
```

Comment:
@aans-odoo  The functional flow & solution fines :+1:  to me but technically, the code related to website module should be in website module only, there is no mean of compiling this lines of code when website module is not installed. I want you to move this code somewhere in website module, may be where the website option is introduced in search bar. This way you will not have to manage the state manually again, there you will have the access of current website & state as well.

Thanks : ) let me know if any thing is unclear here. :bow:

Reply:
Yes, makes sense. I'll make these changes.

Reply:
Yes, makes sense. I'll make these changes.